src/core/pipelines/manifest/create_manifest_pipeline.ts

```diff
@@ -70,6 +70,7 @@ export interface IFetchManifestResult { manifest : Manifest;
  */
 export default function createManifestPipeline(
   pipelines : ITransportPipelines,
+  lowLatencyMode : boolean,
```

Comment:
IMO lowLatency Mode would be better given as a Pipeline option. No need to transport it everywhere like that